templates/build.gradle

```diff
@@ -165,6 +162,17 @@ sourceSets {
     test.java.outputDir = new File("$buildDir/testClasses")
 }
 
+// Adds Resources as parameter for AnnotationProcessor (gather ResourceIndex,
+// also add resource as input for compilejava, for re-gathering ResourceIndex, when resource was changed.
+compileJava { // TODO adds assets
```

Comment:
This should already be including the assets anyway. See the `syncAssets` task below.